docs/docs/reference/new-types/match-types.md

```diff
@@ -90,34 +90,25 @@ Here, `[Xs]` is a type parameter clause of the variables bound in pattern `Pi`.
 `B` is the declared upper bound of the match type, or `Any` if no such bound is given.
 We will leave it out in places where it does not matter for the discussion. Scrutinee, bound and pattern types must be first-order types.
 
+
 ## Match type reduction
 
-We define match type reduction in terms of an auxiliary relation, `can-reduce`:
+Match type reduction follows the semantics of match expression, that is, a match type of the form `S match { P1 => T1 ... Pn => Tn }` reduces to `Ti` if and only if `s: S match { _: P1 => T1 ... _: Pn => Tn }` evaluates to a value of type `Ti` for all `s: S`.
 
-```
-Match(S, C1, ..., Cn)  can-reduce  i, T'
-```
-if `Ci = [Xs] => P => T` and there are minimal instantiations `Is` of the type variables `Xs` such that
-```
-S <: [Xs := Is] P
```

Comment:
The problem is that the rewritten version only talks about subtyping, not about type variable instantiation. But instantiation is crucial for understanding what goes on in more complicated examples. Maybe the existing text is not clear enough. Then it needs to be improved, but it should not be dropped outright.